controller/api/model_controller.go

```diff
@@ -1,11 +1,16 @@
 package api
 
 import (
-	"dummy/mock"
+	"dummy/repository"
 	"encoding/json"
+	"log"
 	"net/http"
 )
 
 func GetModels(w http.ResponseWriter, r *http.Request) {
-	json.NewEncoder(w).Encode(mock.MockModels())
+	models, err := repository.GetModels()
+	if err != nil {
+		log.Fatal(err)
+	}
```

Comment:
Same comment as https://github.com/sungindra/go-dummy-proj/pull/6/files#r1479851050

Reply:
this one wouldn't cause panic, would it?
as this one will encode the nil model as JSON. No attribute access or manipulation happening

Reply:
Have you tried it?